### PR TITLE
add staticArrayToUint8Array

### PR DESCRIPTION
**Pull request by Ben-Rey**

### CODE REVIEW

File: packages/as-types/assembly/__tests__/arrays.spec.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import { staticArrayToUint8Array } from '../serialization';
+
+describe('arrays', () => {
+  test('staticArrayToUint8Array', () => {
+    const myArray = StaticArray.fromArray<u8>([0, 1, 2, 3, 4]);
```
Comment (gregLibert) [inline packages/as-types/assembly/__tests__/arrays.spec.ts:5]:
```suggestion
    const staticArray = StaticArray.fromArray<u8>([0, 1, 2, 3, 4]);
```

File: packages/as-types/assembly/__tests__/arrays.spec.ts [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    const result = staticArrayToUint8Array(myArray);
```
Comment (gregLibert) [inline packages/as-types/assembly/__tests__/arrays.spec.ts:6]:
```suggestion
    const typedArray = staticArrayToUint8Array(myArray);
```

File: packages/as-types/assembly/__tests__/arrays.spec.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+
+    expect(result instanceof Uint8Array).toBe(true);
+    expect(result.length).toBe(myArray.length);
+    for (let i = 0; i < myArray.length; i++) {
+      expect(result[i]).toBe(myArray[i]);
+    }
+  });
```
Comment (gregLibert) [inline packages/as-types/assembly/__tests__/arrays.spec.ts:13]: I would add a etst to confirm that modification is possible:
```suggestion
    typedArray[0] = 10;
    expect(typedArray[0]).toBe(<u8>10);
```

**Review by gregLibert — CHANGES_REQUESTED** [review]

File: packages/as-types/assembly/__tests__/arrays.spec.ts (hunk 3 above)
Comment (Ben-Rey) [reply packages/as-types/assembly/__tests__/arrays.spec.ts:13]: sure

**Review by gregLibert — APPROVED** [review]

**Pull request merged**